app/structures/structures.go

```diff
@@ -21,6 +21,9 @@ type ItemPermissions struct {
 	CanEdit string `json:"can_edit"`
 	// required: true
 	IsOwner bool `json:"is_owner"`
+	// appears only in services that support this permission, and only if it is set
```

Comment:
reading the doc, how do I know if a given service support this permission?

Reply:
fixed, added better doc to be explicit about it. It doesn't feel very satisfying to handle it like this, but the alternative of creating multiple structs just for one field feels worse...